distribution/pom.xml

```diff
@@ -196,9 +196,7 @@
         <inherited>true</inherited>
         <configuration>
           <excludes>
-            <exclude>**/conffiles</exclude>
```

Comment:
Also please drop these lines from `maven-license-plugin` in Drill root POM file:
[`<exclude>**/control</exclude>`](https://github.com/apache/drill/blob/master/pom.xml#L674) and 
[`<conffiles>SCRIPT_STYLE</conffiles>`](https://github.com/apache/drill/blob/master/pom.xml#L701)
and these [files itself](https://github.com/apache/drill/tree/master/distribution/src/deb/control).